lib/trocla/default_config.yaml

```diff
@@ -44,4 +44,12 @@ profiles:
     days: 2
     # 1 day
     expires: 86400
-
+formats:
+  cfssl:
+    server_url: http://localhost:8888
```

Comment:
hmm should we define that at all? Or is thi the default place where your cfssl daemon might be running?

Reply:
default config of the daemon is localhost on port 8888:
```
 cfssl serve --help
  ...
  -address=127.0.0.1: Address to bind
  -port=8888: Port to bind
  ...
```
so I used that as an example